apps/timetable/ui/index.html

```diff
@@ -49,26 +46,16 @@
 
         <div id="gh-modal"><!-- --></div>
 
+        <script id="gh-subheader-part-template" type="text/template">
+            <%= _.partial('subheader-part', {'data': data}) %>
+        </script>
+
         <script id="gh-subheader-picker-template" type="text/template">
-            <option value=""></option>
-            <%_.each(data.courses, function(course) { %>
-                <% if (course.subjects.length === 0) { %>
-                    <option value="<%- course.id %>"><%- course.displayName %></option>
-                <% } else { %>
-                    <optgroup label="<%- course.displayName %>">
-                        <%_.each(course.subjects, function(subject) { %>
-                            <option value="<%- subject.id %>"><%- subject.displayName %></option>
-                        <% }); %>
-                    </optgroup>
-                <% } %>
-            <% }); %>
+            <%= _.partial('subheader-picker', {'data': data}) %>
         </script>
 
-        <script id="gh-subheader-part-template" type="text/template">
-            <option value=""></option>
-            <%_.each(data, function(part) { %>
-                <option value="<%- part.id %>"><%- part.displayName %></option>
-            <% }); %>
+        <script id="gh-subheader-pickers-template" type="text/template">
+            <%= _.partial('subheader-pickers') %>
```

Comment:
Per comment earlier, would it make sense to consolidate this one into the other 2 picker partials?